spec/lib/companies_selector_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'CompaniesSelector' do
+  context 'with query params set (condition default)' do
+    it 'select companies accroding to the params' do
```

Comment:
```suggestion
    it 'selects companies accroding to the params' do
```

The same is true for the rest of the file.

Reply:
Yes, changed, also I changed some more here